tests/php/src/Cli/Export/ExportActivePlugins.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Reference site export active plugins.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Tests\Cli\Export;
+
+use AmpProject\AmpWP\Tests\Cli\ExportStep;
+
+final class ExportActivePlugins implements ExportStep {
+
+	/**
+	 * List of plugin slugs to exclude.
+	 *
+	 * @var string[]
+	 */
+	const EXCLUDED_PLUGINS = [
+		'airplane-mode',
+		'amp',
+		'query-monitor',
+		'redis-cache',
+		'wp-rocket',
```

Comment:
Why these plugins specifically?

Reply:
It's a set of development plugins I'd happen to have installed and want to exclude. This list will probably further grow, but this was what solved it on my end to have a clean list of actual required plugins.